trace/constants.go

```diff
@@ -2,12 +2,28 @@ package trace
 
 //Trace
 const traceType = "Trace"
+const spanType = "Span"
+const traceDataType = "AuditData"
 const executionContext = "ExecutionContext"
 const thundraLambdaTraceRequestDisable = "thundra_lambda_trace_request_disable"
 const thundraLambdaTraceResponseDisable = "thundra_lambda_trace_response_disable"
 
-// Span
-const spanType = "Span"
+//AuditInfo
+const auditInfoOpenTimestamp = "openTimestamp"
```

Comment:
auditInfo related fields are also from the old data model, I think there is a problem with these.